src/components/QuickOpenModal.js

```diff
@@ -390,10 +388,15 @@ export class QuickOpenModal extends Component<Props, State> {
           onChange={this.onChange}
           onKeyDown={this.onKeyDown}
           handleClose={this.closeModal}
-          hasPrefix={this.hasPrefix()}
           expanded={expanded}
           selectedItemId={expanded ? items[selectedIndex].id : ""}
         />
+        {!symbols ||
+          (symbols.functions.length == 0 && (
```

Comment:
sorry, this should be `symbols.identifiers` i think i told you the wrong thing

Reply:
![image](https://user-images.githubusercontent.com/23143862/36407154-6b73c034-15b9-11e8-8e20-5bc57aa47317.png)

These are the only properties of symbols.